pepper-apis/studybuilder-cli/pom.xml

```diff
@@ -49,6 +49,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
```

Comment:
You sure about the `provided` scope? Looks like we are using this scope in all the modules and in the parent.

Reply:
No, I'm not. It's just a copy-paste from another pom file. I will change it to default one - compile